app/components/projectAssignment/workWeekInput.tsx

```diff
@@ -242,7 +242,7 @@ export const WorkWeekInput = ({
 								}
 							}}
 							onFillForwardClick={() => onFillForwardClick(ACTUAL_HOURS, cweek, year, values)}
-							disabled={isUserTBD}
+							disabled={false}
```

Comment:
It caused me error when i try to send data for TBD user, is backend ready for accept hours for TBD user?

Reply:
O I'm glad you caught this this was me simply experimenting with it, the backend at this time was not but I believe fermion has got that figured out.